examples/simple_instance/outputs.tf

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "project_id" {
+  value = "${var.project_id}"
+}
+
+output "subnetwork_project" {
+  value = "${var.subnetwork_project}"
+}
+
+output "subnetwork" {
+  value = "${var.subnetwork}"
+}
+
+output "instance_name" {
+  value = "${var.instance_name}"
+}
+
+output "region" {
+  value = "${var.region}"
+}
+
+output "zone" {
```

Comment:
@morgante This output is piped through to `kitchen-terraform` and used as the `--zone` argument to `gcloud`. Without it, a command like `gcloud --project=berlinsky-container-vm-test compute instances describe simple-instance-vm-test --format json` returns:

```
ERROR: (gcloud.compute.instances.describe) Underspecified resource [simple-instance-vm-test]. Specify the [--zone] flag.
```

Reply:
I started using this pattern on the project-factory, since InSpec's only source of attributes is from Terraform outputs. Some InSpec tests need to know what variables Terraform used in order to set up correct expectations, and this pass through from variables to outputs seemed like the least difficult way of solving this.

Since this code is being used for an example as well as a test case I can see that we don't want to leak test details into examples since it'll be confusing. However this pattern works quite well in non user-facing tests.

Reply:
If we randomly choose a zone that the instance can be deployed in per https://github.com/terraform-google-modules/terraform-google-container-vm/pull/3#discussion_r226307020, then this output makes sense regardless of whether one thinks it makes sense to echo back input.

Reply:
Ok, makes sense.